application-admintools-test/application-admintools-test-pageobjects/src/main/java/com/xwiki/admintools/test/po/AdminToolsHomePage.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.admintools.test.po;
+
+import org.xwiki.test.ui.po.ViewPage;
+
+public class AdminToolsHomePage extends ViewPage
+{
+    public static AdminToolsHomePage gotoPage()
+    {
+        getUtil().gotoPage(getSpace(), getPage());
+        return new AdminToolsHomePage();
+    }
+
+    public static String getPage()
+    {
+        return "WebHome";
+    }
+
+    public static String getSpace()
+    {
+        return "AdminTools";
+    }
+
+    public static String getUrl()
```

Comment:
We use upper case for acronyms.

```suggestion
    public static String getURL()
```

But I don't see where this method is used. In general, we shouldn't add APIs that are not used.